iosApp/Survey/Resources/Assets/Assets.xcassets/Background Blur.imageset/Contents.json

```
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "Image-2.pdf",
```

I think it would be better to match the physical file name with the one in the `Assets.xcassets` 🙏
```suggestion
      "filename" : "Background Blur.pdf",
```